torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -19,22 +20,163 @@
 #include <torch/csrc/jit/codegen/cuda/lower_utils.h>
 #include <torch/csrc/jit/codegen/cuda/lower_validation.h>
 
+// TODO: Move scheduler utils that are useful to ir_utils
+#include <torch/csrc/jit/codegen/cuda/scheduler/utils.h>
+
+#include <list>
+#include <unordered_map>
+#include <unordered_set>
+
 namespace torch {
 namespace jit {
 namespace fuser {
 namespace cuda {
 
 // TODO(kir): revisit this
 thread_local GpuLower* active_gpu_lower = nullptr; // NOLINT
+namespace {
+
+struct idcomp {
+  bool operator()(const IterDomain* lhs, const IterDomain* rhs) const {
+    return lhs->name() < rhs->name();
+  }
+};
```

Comment:
This doesn't seem to be used.

Reply:
Thanks, was stale when I was thinking to actually sort.